homeassistant/helpers/frame.py

```diff
@@ -85,6 +88,12 @@ def report_integration(
     """
     found_frame, integration, path = integration_frame
 
+    # Keep track of integrations already reported to prevent flooding
+    key = f"{found_frame.filename}:{found_frame.lineno}"
+    if key in _REPORTED_INTEGRATIONS:
```

Comment:
If we add this feature, we should remove logic to prevent double warnings from places where this is used.

Reply:
I propose to do that in a follow-up PR